plugin/dns64/README.md

```diff
@@ -6,55 +6,67 @@
 
 ## Description
 
-> DNS64 describes a DNS server that when asked for a domain's AAAA records, but only finds
-> A records, synthesizes the AAAA records from the A records.
+The *dns64* plugins will when asked for a domain's AAAA records, but only finds A records,
+synthesizes the AAAA records from the A records.
 
-The synthesis in only performed if the query came in via IPv6.
+The synthesis is *only* performed **if the query came in via IPv6**.
 
 This translation is for IPv6-only networks that have [NAT64](https://en.wikipedia.org/wiki/NAT64).
 
-See [RFC 6147](https://tools.ietf.org/html/rfc6147) for more information.
-
 ## Syntax
 
+~~~
+dns64 [PREFIX]
+~~~
+
+* **PREFIX** defines a custom prefix instead of the default `64:ff9b::/96`.
+
+Or use this slightly longer form with more options:
+
 ~~~
 dns64 [PREFIX] {
-  [translate_all]
+  translate_all
+  prefix PREFIX
```

Comment:
I prefer the syntax section be not copy-pasteable, to avoid users seeing the first example and always using that. Let's stick to the syntax format.
```suggestion
    [translate_all]
    prefix PREFIX
```